src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $border-color;
```

Comment:
Ensure that the SCSS variable `$border-color` is defined in a separate variables file as per the task requirements.